src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -152,7 +140,7 @@ public bool IsUnitQuaternion
         /// </summary>
         public bool IsRotation
```

Comment:
Maybe we can actually drop this one, seems to be redundant with IsUnitQuaternion?